src/encoded/static/scss/encoded/modules/_pipeline.scss

```diff
@@ -244,7 +244,11 @@ g.pipeline-node-qc-metric {
     @at-root #{&}__data {
         @media screen and (min-width: $screen-md-min) {
             flex: 0 1 33.33%;
+            hr {
+                margin-right: 10px;
```

Comment:
Sorry, I guided you wrong on this one. SCSS nesting can get complicated at times, as in this case where the outer nested `.quality-metrics-modal` gets ignored, so now the hr is too narrow on the file page. I tested a change that works: move the hr out of here and into the @media for `.quality-metrics-modal`, just under the `display: flex`.

Reply:
Thank you very much! There is no way I could figure this out myself.